File: Python/ByTier/B/2166_B_Tab_Closing.py
# Problem: CF 2166 B - Tab Closing
# https://codeforces.com/contest/2166/problem/B

"""
Code Purpose:
This code solves the problem of finding the minimum number of mouse moves required to close all tabs on a screen of length `a`. Each tab has a dynamic length that decreases as tabs are closed, and the cursor starts at the left endpoint. The solution uses mathematical analysis to determine if one move is sufficient or if two moves are needed.

Algorithms/Techniques:
- Mathematical optimization based on comparing `a` and `b`, and the number of tabs `n`
- Greedy approach to minimize mouse moves by calculating how many tabs can be closed in one move

Time Complexity: O(1) - constant time per test case
Space Complexity: O(1) - only using a constant amount of extra space
"""

t = int(input())
for _ in range(t):
    a, b, n = map(int, input().split())
    # If the screen length equals the tab length OR
    # the number of tabs is less than or equal to the number of full-length segments that fit in the screen,
    # then we can close all tabs with just one move
    print(1 if (a == b or n <= a // b) else 2)


# https://github.com/VaHiX/CodeForces/

File: CPP/ByRound/2166/2166_B_Tab_Closing.cpp
// Problem: CF 2166 B - Tab Closing
// https://codeforces.com/contest/2166/problem/B

/*
 * Problem: Tab Closing
 * 
 * Purpose: Given a screen of length `a`, `n` tabs to close, each with a length 
 *          that changes as tabs are closed (len = min(b, a/m)), determine the 
 *          minimum number of mouse moves to close all tabs.
 *          
 *          Tabs are arranged tightly from left to right, with the x's at positions
 *          len, 2*len, ..., m*len. The cursor starts at position 0.
 *          
 *          The key insight is that once we know that all tabs can be covered by 
 *          a single move (i.e., the total space needed is <= a), we only need 1 move.
 *          Otherwise, we have to move at least twice (evenly splitting the tabs).
 * 
 * Algorithms/Techniques: 
 *   - Greedy approach based on interval coverage
 *   - Mathematical analysis to determine minimum moves
 * 
 * Time Complexity: O(T) where T is the number of test cases
 * Space Complexity: O(1)
 */

#include <iostream>

using namespace std;

int main() {
  int T;
  cin >> T;
  while (T--) {
    long long a, b, n;
    cin >> a >> b >> n;
    
    // If b equals a, then each tab has length a, so we can close all
    // tabs in one move regardless of n
    if (b == a || b * n <= a)
      cout << 1 << endl;
    else
      // If tabs are large enough that they cannot all fit in one move,
      // we must make at least two moves to cover all tabs
      cout << 2 << endl;
  }
  return 0;
}


// https://github.com/VaHiX/CodeForces/